monitor/plugins/spark_sahara/plugin.py

```diff
@@ -60,11 +61,8 @@ def __init__(self, app_id, info_plugin, collect_period=2, retries=60, monasca_co
         self.job_ratio = 1.0 / self.number_of_jobs
         self.first_submission_time = None
 
-
     def _publish_measurement(self, job_request):
 
-        time_progress_metric = {}
-        job_progress_metric = {}
```

Comment:
What is the main purpose of this modification (This two deleted variables)? Can you please document this refactor in the description of this PR?

Reply:
This variables is not used in the code.